includes/hub/admin/class-membership-plans-table.php

```diff
@@ -22,25 +22,54 @@ class Membership_Plans_Table extends \WP_List_Table {
 	 */
 	public function get_columns() {
 		$columns = [
-			'id'   => __( 'ID', 'newspack-network' ),
 			'name' => __( 'Name', 'newspack-network' ),
 		];
 		$columns['site_url'] = __( 'Site URL', 'newspack-network' );
 		$columns['network_pass_id'] = __( 'Network ID', 'newspack-network' );
 		if ( \Newspack_Network\Admin::use_experimental_auditing_features() ) {
-			$columns['active_members_count'] = __( 'Active Members', 'newspack-network' );
-			$columns['network_pass_discrepancies'] = __( 'Discrepancies', 'newspack-network' );
+			$columns['active_memberships_count'] = __( 'Active Memberships', 'newspack-network' );
+			$columns['network_pass_discrepancies'] = __( 'Membership Discrepancies', 'newspack-network' );
+
+			$active_subscriptions_sum = array_reduce(
+				$this->items,
+				function( $carry, $item ) {
+					return $carry + ( is_numeric( $item['active_subscriptions_count'] ) ? $item['active_subscriptions_count'] : 0 );
+				},
+				0
+			);
+			$subs_info = sprintf(
+				' <span class="dashicons dashicons-info-outline" title="%s"></span>',
+				__( 'Active Subscriptions tied to this membership plan', 'newspack-network' )
+			);
+			// translators: %d is the sum of active subscriptions.
+			$columns['active_subscriptions_count'] = sprintf( __( 'Active Subscriptions (%d)', 'newspack-network' ), $active_subscriptions_sum ) . $subs_info;
 		}
-		$columns['links'] = __( 'Links', 'newspack-network' );
 		return $columns;
 	}
 
 	/**
 	 * Prepare items to be displayed
 	 */
 	public function prepare_items() {
-		$this->_column_headers = [ $this->get_columns(), [], [], 'id' ];
-		$this->items = Membership_Plans::get_membershp_plans_from_network();
+		$membership_plans_from_network_data = Membership_Plans::get_membership_plans_from_network();
+
+		// Handle table sorting.
+		$order = $_REQUEST['order'] ?? false; // phpcs:ignore WordPress.Security.NonceVerification.Missing, WordPress.Security.NonceVerification.Recommended, WordPress.Security.ValidatedSanitizedInput.InputNotSanitized
+		$orderby = $_REQUEST['orderby'] ?? false; // phpcs:ignore WordPress.Security.NonceVerification.Missing, WordPress.Security.NonceVerification.Recommended, WordPress.Security.ValidatedSanitizedInput.InputNotSanitized
```

Comment:
Can we sanitize these to address the `WordPress.Security.ValidatedSanitizedInput.InputNotSanitized` warning?